revolv/payments/tests/services/test_payment_service.py

```diff
@@ -30,17 +33,35 @@ def test_create_payment(self, mock_amount, mock_instrument):
         )
         payment_instrument = mock.Mock()
         type(payment_instrument).type = instrument_type
+        project = Project.objects.get(id=1)
+        user = RevolvUserProfile.objects.get(id=1)
 
         # Make the payment
         PaymentService.create_payment(
-            self.user,
+            user,
+            user,
             10.00,
+            project,
             payment_instrument,
         )
 
         # Check that the charge was actually made
         payment_instrument.charge.assert_called_once()
 
+        failed = False
+        try:
+            PaymentService.create_payment(
+                user,
+                None,
+                10.00,
+                project,
+                payment_instrument,
+            )
+        except PaymentServiceException:
+            failed = True
```

Comment:
Yeah but I want to make sure that the exception is caught. Won't calling self.fail() result in the entire test failing?